app/controllers/users_controller.rb

```diff
@@ -9,31 +12,24 @@ def index
 
   # GET /users/1
   # GET /users/1.json
-  def show
-  end
+  def show; end
 
   # GET /users/new
   def new
     @user = User.new
   end
 
   # GET /users/1/edit
-  def edit
-  end
+  def edit; end
```

Comment:
- [ ] You [should](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png) be finding the user based on the id in user_params in the edit action